gpii/node_modules/contextManager/test/data/standardMMInput.json

```diff
@@ -22,10 +22,10 @@
                 ],
                 "conditions": [
                     {
-                        "type": "http://gpii.net/common/operators/inRange",
+                        "type": "http://registry.gpii.net/conditions/inRange",
                         "max": 1800,
                         "min": 600,
-                        "inputPath": "http://registry.gpii.net/conditions/timeOfDay"
+                        "inputPath": "http://registry\\.gpii\\.net/environment/luminance"
```

Comment:
... I truly dont understand how they can still be around... I've searched for it and fixed all occurrences - multiple times :) ... anyway, should be fixed now